docs/reference/commandline/search.md

```diff
@@ -18,7 +18,7 @@ keywords: "search, hub, images"
 ```markdown
 Usage:  docker search [OPTIONS] TERM
 
-Search the Docker Hub for images
+Search the Docker registry for images
```

Comment:
You may want to update the actual command to reflect this, as that output will be used to generate the YAML files used to publish the docs; https://github.com/docker/cli/blob/fd060d2f337a2698797836ebfdae74ae2e604869/cli/command/registry/search.go#L35

Reply:
Looks like the `index.md` also needs to be updated; https://github.com/docker/cli/blame/704d37db8f545f60c7dcea8c3677f63d96ef1db0/docs/reference/commandline/index.md#L92

One thing worth noting is that (afaik), `Docker Hub` is currently hard-coded, so I wonder if users may confuse "Docker registry" with their own Docker local registry?